# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: netwerk/protocol/http/HttpConnectionBase.h
# Commit: 521c9e019678
# Full Hash: 521c9e019678f584590f63473416d40e03f33342
# Author: Valentin Gosu <valentin.gosu@gmail.com>
# Date: 2023-03-15 04:10:55
# Regressor Bug: 1805372
# File Overlap Count: 1
# Description:
#   Bug 1805372 - Pass effectiveTRRMode to nsHttpChannel from socketTransport/DnsAndConnectSocket r=necko-reviewers,kershaw
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D164539
# ==============================================================================

diff -r dfb39ca05529 -r 521c9e019678 netwerk/protocol/http/HttpConnectionBase.h
--- a/netwerk/protocol/http/HttpConnectionBase.h	Tue Mar 14 15:29:49 2023 +0200
+++ b/netwerk/protocol/http/HttpConnectionBase.h	Tue Mar 14 12:59:40 2023 +0000
@@ -143,6 +143,8 @@
   virtual nsresult GetSelfAddr(NetAddr* addr) = 0;
   virtual nsresult GetPeerAddr(NetAddr* addr) = 0;
   virtual bool ResolvedByTRR() = 0;
+  virtual nsIRequest::TRRMode EffectiveTRRMode() = 0;
+  virtual TRRSkippedReason TRRSkipReason() = 0;
   virtual bool GetEchConfigUsed() = 0;
   virtual PRIntervalTime LastWriteTime() = 0;
 